docs/publish.md

```diff
@@ -0,0 +1,9 @@
+# Publishing to zapier for Kontent.ai developers
+
+1. Even though the zapier-cli is in devDependencies you need to install it globally to be able to login. 
+    -  `npm install -g zapier-platform-cli`
+1. Use `zapier login` and login with corresponding credentials
+1. This will create a file `.zappierrc` somewhere in `C:\Users\{user}`
+1. Login to https://developer.zapier.com/, click on your avatar and open settings. Navigate to deploy keys and copy it into your clipboard.
+1. Open `.zappierrc` with your favourite editor and change the deploy key here with the one in your clipboard.
```

Comment:
Is this step necessary? I managed to push without doing that.

Reply:
It's kind of weird. When I tried logging in for the first time, it did not create a new deploy key in the account settings on Zapier. But when I logged out, then logged in again, it added the key and pushed successfully.